fastly/block_fastly_service_v1_gzip.go

```diff
@@ -127,6 +127,7 @@ func (h *GzipServiceAttributeHandler) Process(d *schema.ResourceData, latestVers
 		// this and so we've updated the below code to convert the type asserted
 		// int into a uint before passing the value to gofastly.Uint().
 		if v, ok := modified["content_types"]; ok {
+			opts.ContentTypes = gofastly.String("")
```

Comment:
Might be worth adding a comment here (and same for `opts.Extensions = gofastly.String("")`) to state that this line was added to address a change in the backend API where it used to accept an empty value but now will use a default value if no value provided, so to avoid a diff we always default to sending an empty string.